specs/requests/artifacts_spec.py

```diff
@@ -102,6 +112,32 @@
                 with it("returns an empty list for tags"):
                     expect(self.response.json.get("tags")).to(equal([]))
 
+                with it("saves the image to the configured uploads directory"):
+                    file_name = self.response.json.get("url").split("/")[-1]
+                    file_path = os.path.join(current_app.config["UPLOAD_FOLDER"], file_name)
```

Comment:
You are replicating a lot of application logic in these tests.
The file_name should not be retrieved from the response imo. And the file_path should be hardcoded as well.

Reply:
The problem is that I don't know the file name if I don't retrieve it from the response :(
The backend saves it on a very different name so there is no chance to find the file without checking the response.

Reply:
Why hard code the file_path when the upload folder is configurable. It would just break the tests if you use a rename it.